docs/api/instruments/validators.rst

```diff
@@ -4,7 +4,7 @@
 Validator functions
 ###################
 
-Validators are used in conjunction with the :func:`Instrument.control <pymeasure.instruments.Instrument.control>` function to allow properties with complex restrictions for valid values. They are described in more detail in the :ref:`Advanced properties <advanced-properties>` section.
+Validators are used in conjunction with the :func:`Instrument.control <pymeasure.instruments.Instrument.control>` function to allow properties with complex restrictions for valid values. They are described in more detail in the :ref:`validators` section.
```

Comment:
I think validators can be also used with `setting` function, which is a particular use case of `control`.